### DISCUSSION TITLE
All coins stolen

### DISCUSSION THREAD

**Discussion by vesphp** [Chia-Network/chia-blockchain#14745] [Support]
Yesterday, all the coins were stolen from my cold wallet.

https://xchscan.com/en/address/xch1ajh4j0fzyzj6qxkxv3g67u2y0s84qnxyyx9lyvz7w6f6ny5gp6usu8vjj3

How could this happen?

**Comment by gtwins1980** [comment]
我的也被偷了，check the address ：   xch16euvttx9ynd68xzl0qefc6k2jahycuwzr7umlqya4z4hq57r2dkq6tdugu

**Reply by gtwins1980** [reply]
This address has stolen 230,598.0358475648 XCH

**Comment by bellinghamy** [comment]
They want more money in and when you decline to take loans worth of €90,000.00 they’ll empty your investment account. That’s what happened to me. In the beginning it seemed to be ok, but when you want to take money out it is not possible. They say it takes long time to gather the money etc etc. There’s always excuse why you can not have your money back. I have contacted them via customer service and I never got reply in written. My first contact to them was the trading pause of seven weeks. I had that time on my account €85,654.00, suddenly when they started to do the trading, there was less that €20,867.00 left. I checked the trading history win loss report and there wasn’t loss enough that could’ve explained the disappearance of 60,567.0 euros. I was not able to get access to my invested capital including my bonus in the Bloombex account, God so kind, luckily as I was researching for how to get my funds back from the scam broker, then I came across a comment on the broker reviews website, that talks about a recovery expert, on binary,forex,crypto and bitcoin trade recovery, they were to guide me through the process to get my funds from the scam investment broker Bloombex option trade. Him and his team worked relentlessly to get my refund.

Feel free to reach out to him , if you had same experience via  email FASTFUNDSRECOVERY.NET

**Comment by korvin970** [comment]
good afternoon dear bot chia network. My CHIA wallet was robbed of all the XCH coins in there. the coins are still in the thief's wallet. I can prove the origin of these coins. The entire chain of acquisition and input to the wallet. I request you to investigate this matter. And if possible, return my tokens to me

**Comment by BrandtH22** [comment]
Hello @vesphp  , I am sorry to hear that the coins were stolen and urge you to review our article regarding how to be a hard target:
https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target/ .

The chia blockchain is fully decentralized meaning we at Chia Network do not have any ability to return stolen or lost funds and are unable to assist in the recovery of funds.

**Comment by BrandtH22** [comment]
@vesphp , unfortunately it appears someone has gained access to your keyset and/or device and stolen your funds. Since chia is a truly decentralized network there is no way to reclaim the funds or revert the transaction(s). One thing we recommend is ensuring that you and your farm are following security best practices to reduce malicious actors ability to gain access to your keys and/or device, many of these best practices are outlined here: https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target/

Since we unfortunately cannot do more to assist with the stolen funds we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
